### PR TITLE
Add a link to Python versions in smoke test docs & update how to build docs

### PR DESCRIPTION
**Pull request by edwardchalstrey1**
## :white_check_mark: Checklist

- [x] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [x] You are targeting the **develop branch**.
- [x] Your branch is up-to-date with the **develop branch** (you probably started your branch from `develop` but it may have changed since then).

### :arrow_heading_up: Summary

Add a link to Python versions in smoke test docs & how to build the docs

### CODE REVIEW

**Comment by edwardchalstrey1** [comment]
As far as I can tell, the actions failures are unrelated to the changes - apart from the md link to docs/README failing because it's not on the develop branch yet

**Review by jemrobinson — APPROVED** [review]
LGTM

**Comment by JimMadge** [comment]
```
* At docs/_output/README.html:390:

  External link https://github.com/alan-turing-institute/data-safe-haven/edit/develop/docs/README.md failed (status code 404)
```

It looks like an html docs page for the README is being generated, which I assume isn't intentional.

Can we make Sphinx ignore `README.md`?

The other links appear to still be live, so that might be an html-proofer problem. If they continue to fail it would be worth specifically excluding those URLs.

The powershell failure might be best solved in another PR.

**Review by JimMadge — CHANGES_REQUESTED** [review]
Comments above

**Comment by edwardchalstrey1** [comment]
> ```
> * At docs/_output/README.html:390:
> 
>   External link https://github.com/alan-turing-institute/data-safe-haven/edit/develop/docs/README.md failed (status code 404)
> ```
> 
> It looks like an html docs page for the README is being generated, which I assume isn't intentional.
> 
> Can we make Sphinx ignore `README.md`?
> 
> The other links appear to still be live, so that might be an html-proofer problem. If they continue to fail it would be worth specifically excluding those URLs.
> 
> The powershell failure might be best solved in another PR.

@JimMadge I've moved the info to the main README which makes it more visible as well as avoiding this issue

File: README.md [hunk 1]
```diff
@@ -44,6 +44,20 @@ Read our [versioning scheme](VERSIONING.md) for how we number and label releases
...
+
+To build the docs locally, check out the repo, navigate to the `docs` folder and `make` them:
+
+```{bash}
+git clone https://github.com/alan-turing-institute/data-safe-haven.git
+cd data-safe-haven/docs
+make html
+```
+
+This will add the contents to a folder called `_output` inside `docs`.
```
Comment (jemrobinson) [inline README.md]: It's worth pointing to the version of hosted on github.io but I'm not sure whether people who are viewing the front page of our repo will be interested in instructions about how to build a local copy of the docs.

Comment (edwardchalstrey1) [reply README.md]: They might do if they want to PR a doc fix maybe? I could even say: "If you notice something looks wrong in the docs, fork, then clone, then modify and test with make, then PR" - but perhaps that should live in [contributing](https://github.com/alan-turing-institute/data-safe-haven/blob/develop/CONTRIBUTING.md)?

Comment (jemrobinson) [reply README.md]: Yes, probably somewhere in the contributing guide would be best.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,11 @@ There are lots of ways to contribute, not just writing code!
 
 See our [Code of Conduct](CODE_OF_CONDUCT.md) and our [Contributor Guide](CONTRIBUTING.md) to learn more about how we work together as a community and how you can contribute.
 
+## :book: Documentation
+
+Full documentation is available at the [Data Safe Haven website](https://alan-turing-institute.github.io/data-safe-haven).
+If you think anything there is unclear, please consider reading how to contribute a documentation change in our [Contributor Guide](CONTRIBUTING.md#making-a-change-to-documentation).
```
Comment (jemrobinson) [inline README.md]: Do we actually need this section at all? The docs are already linked from the `Community & support` section above, while how to contribute is covered in the `Contributing` section.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ Jump to the following sections:
     - [Working in a private repository](#working-in-a-private-repository)
     - [Who's involved in the project](#whos-involved-in-the-project)
     - [Make a change with a pull request](#making-a-change-with-a-pull-request)
+    - [Make a change to documentation](#making-a-change-to-documentation)
```
Comment (JimMadge) [inline CONTRIBUTING.md]:
```suggestion
    - [Make a change to the documentation](#making-a-change-to-the-documentation)
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -219,6 +220,21 @@ You can update your [fork](https://docs.github.com/en/get-started/quickstart/for
 
 GitHub has a [nice introduction](https://docs.github.com/en/get-started/quickstart/github-flow) to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions :balloon:.
 
+### Making a change to documentation
```
Comment (JimMadge) [inline CONTRIBUTING.md]:
```suggestion
### Making a change to the documentation
```

**Review by JimMadge — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (edwardchalstrey1) [reply README.md]: I'm happy to remove it

**Review by JimMadge — APPROVED** [review]

**Pull request merged**